systems/plants/RigidBodyManipulator.cpp

```diff
@@ -248,7 +248,7 @@ void getFiniteIndexes(T const & v, std::vector<int> &finite_indexes)
 {
   finite_indexes.clear();
   const size_t n = v.size();
-  for (size_t x = 0; x < n; x++)
+  for (int x = 0; x < n; x++)
```

Comment:
Does MSVC not like initializing size_t in for loops?  The loop condition is comparing to a size_t.